example/plugins/c-api/thread_pool/test/SDKTest/psi_server.c

```diff
@@ -68,7 +69,7 @@ int
 generate_psibility()
 {
   double rand;
-  rand = drand48();
+  rand = ts::Random::drandom();
```

Comment:
not sure if I should edit this c file (I don't know how to run it either

Reply:
This file looks abandoned for a long time. We can just replace it for now.
https://github.com/apache/trafficserver/issues/8561